test/withStyles_test.jsx

```diff
@@ -196,7 +198,12 @@ describe('withStyles()', () => {
       }))(MyComponent);
 
       // copied
-      expect(Wrapped.propTypes).to.eql(MyComponent.propTypes);
+      const expectedPropTypes = deepmerge({}, MyComponent.propTypes);
+      delete expectedPropTypes.styles;
+      delete expectedPropTypes.theme;
+      expect(Wrapped.propTypes).to.eql(expectedPropTypes);
+      expect(MyComponent.propTypes).to.include.keys('styles', 'theme');
```

Comment:
it might be useful to also add an assertion for `expect(Wrapped.propTypes).not.to.include.keys('styles', 'theme');`, before the `eql`?